lxd/storage/drivers/driver_dir.go

```diff
@@ -390,6 +391,16 @@ func (d *dir) CreateVolumeFromMigration(vol Volume, conn io.ReadWriteCloser, vol
 	err = vol.MountTask(func(mountPath string, op *operations.Operation) error {
 		path := shared.AddSlash(mountPath)
 
+		// Run the volume pre-filler function if supplied.
+		if preFiller != nil && preFiller.Fill != nil {
```

Comment:
@stgraber the image is unpacked first before syncing the snapshots or parent volume. This seems like the right thing to do as the snapshots should contain most of the base image too.

Reply:
Yep, that sounds right